doc/modules/ROOT/pages/index.adoc

```diff
@@ -73,21 +74,26 @@ Our application with Hazelcast caching is now ready to go. We do not need to cha
 
 Build the application by using maven, make sure you are in the project directory
 
-[source]
+[source,shell]
 ----
 mvn clean package
 ----
 
+[NOTE]
+====
+The very first request to each application instance will take a bit longer time to respond since the first request causes hazelcast instance to be started.
```

Comment:
nit
```suggestion
The very first request to each application instance will take a bit longer time to respond since the first request causes Hazelcast instance to be started.
```